core/src/cluster_info_vote_listener.rs

```diff
@@ -37,22 +46,25 @@ impl ClusterInfoVoteListener {
         cluster_info: &Arc<RwLock<ClusterInfo>>,
         sigverify_disabled: bool,
         sender: &Sender<VerifiedPackets>,
+        poh_recorder: Arc<Mutex<PohRecorder>>,
     ) -> Result<()> {
         let mut last_ts = 0;
         loop {
             if exit.load(Ordering::Relaxed) {
                 return Ok(());
             }
             let (votes, new_ts) = cluster_info.read().unwrap().get_votes(last_ts);
```

Comment:
why is this call outside the check?  looks like we just drop those votes each time...